src/plugins/intel_npu/src/backend/src/zero_init.cpp

```diff
@@ -20,73 +20,38 @@ namespace intel_npu {
 
 const ze_driver_uuid_t ZeroInitStructsHolder::uuid = ze_intel_npu_driver_uuid;
 
-static std::tuple<uint32_t, std::string> queryDriverExtensionVersion(
-    std::vector<ze_driver_extension_properties_t>& extProps,
-    uint32_t count) {
-    // query the extension properties
-    const char* graphExtName = nullptr;
-    uint32_t targetVersion = 0;
-    for (uint32_t i = 0; i < count; ++i) {
-        auto& property = extProps[i];
-
-        if (strncmp(property.name, ZE_GRAPH_EXT_NAME, strlen(ZE_GRAPH_EXT_NAME)) != 0) {
-            continue;
-        }
-
-        // If the driver version is latest, will just use its name.
-        if (property.version == ZE_GRAPH_EXT_VERSION_CURRENT) {
-            graphExtName = property.name;
-            targetVersion = property.version;
-            break;
-        }
-
-        // Use the latest version supported by the driver.
-        if (property.version > targetVersion) {
-            graphExtName = property.name;
-            targetVersion = property.version;
-        }
-    }
-
-    if (graphExtName == nullptr) {
-        OPENVINO_THROW("queryDriverExtensionVersion: Failed to find Graph extension in NPU Driver");
-    }
-
-    const uint16_t supportedDriverExtMajorVersion = 1;
-    const uint16_t driverExtMajorVersion = ZE_MAJOR_VERSION(targetVersion);
-    if (supportedDriverExtMajorVersion != driverExtMajorVersion) {
-        OPENVINO_THROW("Plugin supports only driver with extension major version ",
-                       supportedDriverExtMajorVersion,
-                       "; discovered driver extension has major version ",
-                       driverExtMajorVersion);
-    }
-
-    return std::make_tuple(targetVersion, graphExtName);
-}
-
-static uint32_t queryMutableCommandListVersion(std::vector<ze_driver_extension_properties_t>& extProps,
-                                               uint32_t count) {
-    // query the mutable command list properties
+static std::tuple<uint32_t, std::string> queryExpVersion(const char* extName,
```

Comment:
Where does "exp" come from? Most probably "ExtVerion" ?

Reply:
And having "driver" in the name helped to express that we are querying the version supported by the driver, while we can have a different version supported by the plugin.

Reply:
done